devops/TestCrossSiteScripting/testCrossSiteScripting.py

```diff
@@ -0,0 +1,108 @@
+import requests
+import argparse
+
+# main driver function that identifies and deletes SageMaker stacks
```

Comment:
A typo?

Reply:
How did you fix that Import? I am not able to run it.
![image](https://user-images.githubusercontent.com/72327494/158706197-9b50c92e-7b42-436c-9833-8f5377e93103.png)

Reply:
You need to import the library with: `python -m pip install requests`

Reply:
> A typo?

Good catch, I will update that.

Reply:
Thanks Nate.Workes as expected!